X/Xorg_libXt/build_tarballs.jl

```diff
@@ -3,21 +3,24 @@
 using BinaryBuilder
 
 name = "Xorg_libXt"
-version = v"1.2.0"
+version = v"1.3.1"
 
 # Collection of sources required to build libXt
 sources = [
-    ArchiveSource("https://www.x.org/archive/individual/lib/libXt-$(version).tar.bz2",
-                  "b31df531dabed9f4611fc8980bc51d7782967e2aff44c4105251a1acb5a77831"),
+    ArchiveSource("https://www.x.org/archive/individual/lib/libXt-$(version).tar.xz",
+                  "e0a774b33324f4d4c05b199ea45050f87206586d81655f8bef4dba434d931288"),
+    DirectorySource("bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libXt-*/
-CPPFLAGS="-I${prefix}/include"
-# When compiling for things like ppc64le, we need newer `config.sub` files
-update_configure_scripts
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-malloc0returnsnull=no
+cd $WORKSPACE/srcdir/libXt-*
+
+# Correct syntax error in header file
+# (Already fixed upstream.)
```

Comment:
What does this mean?  Is there a new version available?  If not, how's possible they released such a broken version without a quick followup?

Reply:
It's fixed on the master branch but not yet released. The fix look exactly like you think it would. We don't need to submit a PR to upstream because they fixed it, but if we want to stick to a released version, then we should apply a patch.